app/models/organization.rb

```diff
@@ -8,6 +8,8 @@ class Organization < ApplicationRecord
   has_many :billing_histories, dependent: :destroy
   has_many :collections, dependent: :destroy
   has_many :organization_users, dependent: :destroy
+  has_many :organization_emails, dependent: :destroy
+  has_one :theme, dependent: :destroy
```

Comment:
there is no existance of theme, please delete

Reply:
already fixed it in coming PRs forgot to change in this one